tsconfig.json

```diff
@@ -1,15 +1,30 @@
 {
-  "compileOnSave": false,
+  "compileOnSave": false,  
   "compilerOptions": {
+    "declaration": true,
     "downlevelIteration": true,
     "target": "ES6",
     "module": "ES2015",
     "moduleResolution": "node",
-    "allowJs": true,
+    "allowJs": false,
     "importHelpers": true,
+    "allowSyntheticDefaultImports":  true,
     "strict": true,
     "baseUrl": ".",
-    "noEmit": true,
+    "noEmit": true, //Deactivate "noEmit" to generating d.ts-File to 'builds/dts/dist' via "make dts"
+    "emitDeclarationOnly": true,
+    "outDir": "builds/dts/dist",
```

Comment:
I feel like the builds/dts/dist may change depending on whether it's the knockout or the tko build, but also it's ok for now - just noting.

Reply:
This is just an poc for now. The normal build chain is not affected by these parameters. I'm still unclear on how best to build a typing package. Once the PR is complete, I'd like to try it with https://www.npmjs.com/package/esbuild-plugin-d.ts

Regarding the build pipeline, I would preferred something like a one file bundle for TKO and @types/tko, with an esm export: `export var tko`. Corresponds to browser.es6.js

Reply:
Thanks, I'm not clear on the best build strategy either.  It'd be nice if the only dependency were Typescript itself, for the typing.

On the @types/tko we _should_ be ok, noting that the static global for the knockout and the tko build may be different, so we want to be mindful of that.